enviroment.yaml

```diff
@@ -0,0 +1,13 @@
+channels:
+  - conda-forge
+  - bioconda
+  - defaults
+dependencies:
+  - python=3.9.*
+  - pandas=1.4.*
```

Comment:
We should probably bump the versions of these dependencies before pushing to master in future (e.g., pandas could now be bumped to pandas 2)

Reply:
Added as issue